--- pb.py.orig	2008-04-10 13:52:17.000000000 +0100
+++ pb.py	2008-04-10 13:52:03.000000000 +0100
@@ -441,6 +441,13 @@
     printBriefTraceback = printTraceback
     printDetailedTraceback = printTraceback
 
+    def setCopyableState(self, state):
+        klass = reflect.namedAny(state['type'])
+        inst = klass(state['value'])
+        state['value'] = inst
+        state['type'] = klass
+        self.__dict__ = state
+
 setUnjellyableForClass(CopyableFailure, CopiedFailure)
 
 def failure2Copyable(fail, unsafeTracebacks=0):
